ResearchKit/Common/ORKMultipleValuePicker.m

```diff
@@ -251,6 +253,30 @@ - (NSString *)pickerView:(UIPickerView *)pickerView titleForRow:(NSInteger)row f
     }
 }
 
+- (UIView *)pickerView:(UIPickerView *)pickerView viewForRow:(NSInteger)row forComponent:(NSInteger)component reusingView:(UIView *)view
+{
+    UILabel* valueLabel = (UILabel*)view;
+    if (!valueLabel)
+    {
+        valueLabel = [[UILabel alloc] init];
+        [valueLabel setFont:[self defaultFont]];
+        [valueLabel setTextAlignment:NSTextAlignmentCenter];
+    }
+    valueLabel.text = [self pickerView:pickerView titleForRow:row forComponent:component];
+    return valueLabel;
```

Comment:
I just did this for Sage's [shared research framework](https://github.com/ResearchKit/SageResearch) and discovered that centering all the rows will result in really weird looking UI under some circumstances (eg. blood pressure, duration formatted for hours/minutes/seconds, month/year date entry, etc.). What we ended up doing was computing the size of all the labels, caching the sizes, and then special-casing the first and last components. Then the first component is formatted to right alignment and the last component is formatted to left alignment. Only the components in the center are center-aligned. We used calculated widths b/c if the picker is ever used for something like month/year (which is not supported by UIDatePicker), then the widths will not be even. We considered optimizing the code by using character count, etc (because calculated text bounds are expensive) but decided that doing so would be premature optimization.

https://github.com/ResearchKit/SageResearch/blob/master/ResearchSuiteUI/ResearchSuiteUI/iOS/Views/RSDPickerViewProtocol.swift

Reply:
@syoung-smallwisdom the default alignment, prior to this PR, is center so I just implemented the same.

I agree with you that on some use cases changing the alignments will become necessary which can be done by the developer as they see fit for their exact application.